Mathlib/CategoryTheory/Pi/Monoidal.lean

```diff
@@ -0,0 +1,45 @@
+/-
+Copyright (c) 2025 Sina Hazratpour. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sina Hazratpour
+-/
+
+import Mathlib.CategoryTheory.Pi.Basic
+import Mathlib.CategoryTheory.Monoidal.Category
+
+/-!
+# The pointwise monoidal structure on the product of families of monoidal categories
+
+Given a family of monoidal categories `C i`, we define a monoidal structure on
+`Π i, C i` where the tensor product is defined pointwise.
+
+-/
+
+namespace CategoryTheory
+
+namespace Pi
+
+open Category MonoidalCategory
+
+universe w₁ v₁ u₁
+
+variable {I : Type w₁} (C : I → Type u₁) [∀ i, Category.{v₁} (C i)]
+[∀ i, MonoidalCategory (C i)]
+
+/-- `Pi.monoidal C` equips the product of an indexed family of categories with
+the pointwise monoidal structure. -/
+@[simps]
```

Comment:
`simps` doesn't generate all the lemmas that would be useful during manual rewrites. Could you add
```lean
theorem associator_hom_apply {X Y Z : ∀ i, C i} {i : I} :
    (α_ X Y Z).hom i = (α_ (X i) (Y i) (Z i)).hom := rfl

theorem isoApp_associator {X Y Z : ∀ i, C i} {i : I} :
    isoApp (α_ X Y Z) i = α_ (X i) (Y i) (Z i) := rfl
```
and so on, also for the inverse of the associator, and the left and right unitor?